libs/full/runtime_distributed/src/server/runtime_support_server.cpp

```diff
@@ -356,8 +355,6 @@ namespace hpx { namespace components { namespace server {
             // Note: we protect the entire loop here since the stopping condition
             // depends on the shared variable "dijkstra_color_"
             // Proper unlocking for possible remote actions needs to be taken care of
```

Comment:
This comment is outdated if the lock will be removed.

Reply:
Will remove it.